crossdock/client/trace.go

```diff
@@ -161,3 +161,15 @@ func transport2transport(v string) tracetest.Transport {
 		panic("Unknown protocol " + v)
 	}
 }
+
+func compareTraceIDs(id1, id2 string) bool {
```

Comment:
my two cents: to me `compare` implies a ternary result (-1: less than, 0: equals, 1: greater than). Whereas I think `equal` or `areEqual` would be better suited to a binary result.